components/gc-toc/toc-en.html

```diff
@@ -0,0 +1,57 @@
+---
+{
+	"title": "GCWeb Table of Contents example",
+	"language": "en",
+	"altLangPage": "toc-fr.html",
+	"breadcrumbs": [
+		{ "title": "GCWeb home", "link": "https://wet-boew.github.io/GCWeb/index-en.html" }
+	],
+	"dateModified": "2023-06-16"
+}
+---
+<div class="wb-prettify all-pre"></div>
+
+<h2 id="toc">In-page table of contents</h2>
```

Comment:
Don't need this heading because it duplicate the content of the heading level 1. And bump the sub-heading one level up